tests/suites/0_stateless/00_0000_dummy_select_1.sql

```diff
@@ -7,3 +7,4 @@ SELECT NOT(1=1) from numbers(3);
 SELECT TRUE;
 SELECT FALSE;
 SELECT NOT(TRUE);
+SELECT a.number FROM numbers(3) AS a order by a.number;
```

Comment:
What will happen?
``` sql
SELECT b.number FROM numbers(3) AS a ORDER BY a.number;
SELECT a.number FROM numbers(3) AS a ORDER BY b.number;
SELECT b.number FROM (SELECT * FROM numbers(3) AS b ORDER BY a.number) AS a;
SELECT b.number FROM (SELECT * FROM numbers(3) AS b) AS a ORDER BY b.number;
```

Reply:
All above four statement has similar issue like:

`ERROR 1105 (HY000): Code: 25, displayText = Unknown Table '"b"'.`

Reply:
👍 Maybe we add them to the test?

Reply:
Added.  PTAL @zhang2014